integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/CarbonDescribeFormattedCommand.scala

```diff
@@ -65,6 +65,7 @@ private[sql] case class CarbonDescribeFormattedCommand(
     val dims = relation.metaData.dims.map(x => x.toLowerCase)
     var results: Seq[(String, String, String)] = child.schema.fields.map { field =>
       val fieldName = field.name.toLowerCase
+      val colComment = field.getComment().getOrElse("null")
```

Comment:
Also needs to make sure that "desc table" also works. As spark/Hive supports accordingly, need to make it consistant

Reply:
verified, it is in sync with hive, if comment is not there, by default it will be null